File: main/src/main/java/net/fwitz/math/main/binary/split/functions/examples/simple/SplitInverseInverse.java
package net.fwitz.math.main.binary.split.functions.examples.simple;

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitInverseInverse {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("1 / (1/z)")
                .computeFn(z -> z.inverse().inverse())
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/examples/simple/SplitInverseInverse.kt
package net.fwitz.math.main.binary.split.functions.examples.simple

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitInverseInverse {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("1 / (1/z)")
        .computeFn { z -> z.inverse.inverse }
        .render()
}